docs/dictionaries/arWordlistReadme.txt

```diff
@@ -0,0 +1,5 @@
+Source: https://github.com/a3f/arabic-wordlists
```

Comment:
Wikipedia is not really a good word source if used on its own. It may include a lot of technical terms, but it may lack commonly used everyday words.

Nevertheless, This is not a problem. There are many other free sources, like [this one](https://github.com/jamalamch/Arabic-Words/blob/master/AllListWordArabic.txt). The author claims to have gathered all possible words, and it is indeed an extensive list. I can include them before merging.

Reply:
I updated and briefly looked over the arabic word list you have recommended and it looks good, i left it in the format that it was downloaded and just renamed the txt file and updated the yml file to point to it, i have also updated the arWordlistReadme to reflect the new list

Reply:
I marked this conversation as resolved for now because I dont see any more work left to do for the dictionary files, please feel free to unresolve it if something comes up.